velox/exec/Task.cpp

```diff
@@ -1507,6 +1507,41 @@ ContinueFuture Task::requestPauseLocked(bool pause) {
   return makeFinishFutureLocked("Task::requestPause");
 }
 
+// Checks if it is likely that the reservation on 'tracker' can be
+// incremented by 'increment'. Returns false if this seems
+// unlikely. Otherwise attempts the reservation increment and
+// returns true if succeeded. This will be moved to Task when adding
+// memory contention arbitration.
+// static
+bool Task::maybeReserve(
```

Comment:
Why is this method on the Task and not on MemoryUsageTracker? It doesn't use anything from the Task.